### ISSUE TITLE
[plot] Colormap witout vmin/vmax

### ISSUE THREAD

**Issue by vallsv** [silx-kit/silx#915]
On pyFAI we are setting a colormap like that:
```
colormap = {
    'name': "inferno",
    'normalization': 'log',
    'autoscale': True,
}
self.__plot2D.setDefaultColormap(colormap)
```
It was working since the last master https://github.com/silx-kit/pyFAI/issues/615.
I can fix the master of pyFAI but i can't fix the previous version of pyFAI.
For the next release of silx it is import to check that this API is working as pyFAI 0.13.1 uses it.

**Comment by t20100** [comment]
@vallsv, I've tested it and it seems to be fixed since PR #898

**Comment by vallsv** [comment]
Looks to be fixed
